queue/storage.go

```diff
@@ -0,0 +1,46 @@
+package queue
```

Comment:
@ogbanugot Do we need to copy this guy? I think we can keep the interface but get `taskq.NewLocalStorage()`

Please correct me if I'm wrong.